### DISCUSSION TITLE
Forward/backward in music mode

### DISCUSSION THREAD

**Discussion by kathytu** [iina/iina#4716] [Q&A] [answered]
Hey all, I can't figure this out, is it possible to change the next/previous track buttons in Music Mode to be seek forward/backward 10s? Am I missing that option somewhere in settings? Or does it not currently exist?
![CleanShot 2023-12-15 at 14 35 36@2x](https://github.com/iina/iina/assets/5014981/b23e15f1-7665-4a3d-a063-29d031c92463)

Thank you!

**Comment by svobs** [comment, marked as answer]
No it is not currently possible in Music Mode.

In "regular" mode, you can use this setting:
<img width="588" alt="SCR-20231215-qkem" src="https://github.com/iina/iina/assets/2213815/bb0a4215-424a-4236-80ac-69ad89e46e7f">

But that is not currently honored in Music Mode. Instead the buttons are hardwired to "playlist prev" "playlist next". 

This would be a good feature to file a request for.
